### ISSUE TITLE
Window does not fit to viewport size

### ISSUE THREAD

**Issue by devzerker** [playwright-community/jest-playwright#268]
**Describe the bug**
In headfull mode window does not fit to viewport size. See the screenshots below

**To Reproduce**
1. Take ready example from [here](https://github.com/playwright-community/playwright-jest-examples/tree/master/basic-ts)
2. In package.json file change jest-playwright-preset version to 1.3 or stay on 1.2.1
3. In jest-playwright.config.js file change headless to false and use default context options or set custom, for example
 ```js
module.exports = {
    launchOptions: {
        headless: false
    }
};
```
 ```js
module.exports = {
    launchOptions: {
        headless: false
    },
    contextOptions: { viewport: { width: 1600, height: 900 } }
};
```
 ```js
module.exports = {
    launchOptions: {
        headless: false
    },
    contextOptions: { ...require('playwright').devices['iPhone 11 Pro'] }
};
```
4. Install packages and run tests
 ```
npm i
```
 ```
npm run test
```

**Screenshots**
Default viewport:
![image](https://user-images.githubusercontent.com/10089615/89105258-48009700-d428-11ea-8289-980e4af675cf.png)
1600x900:
![image](https://user-images.githubusercontent.com/10089615/89105316-be9d9480-d428-11ea-871b-06f4db4c270c.png)
iPhone11:
![image](https://user-images.githubusercontent.com/10089615/89105269-564eb300-d428-11ea-9c4b-f49ff1a37c3d.png)

**Expected behavior**
Window fits to viewport size. Screenshots from clean playwright:
Default viewport:
![image](https://user-images.githubusercontent.com/10089615/89105504-5fd91a80-d42a-11ea-9d3f-4555fdbfbb4f.png)
1600x900:
![image](https://user-images.githubusercontent.com/10089615/89105519-8008d980-d42a-11ea-85ea-02626ca3cba5.png)
iPhone11:
![image](https://user-images.githubusercontent.com/10089615/89105558-c2321b00-d42a-11ea-8993-8deb238f2f81.png)

**Desktop:**
 - OS: Windows 10
 - Node version 14.7.0
 - Playwright version 1.2.1
 - jest-playwright version either 1.2.1 or 1.3.0

**Jest configuration:**

 ```js
module.exports = {
  verbose: true,
  preset: 'jest-playwright-preset',
  transform: {
    '^.+\\.ts$': 'ts-jest',
  },
}
```

**Comment by mmarkelov** [comment]
@devzerker I'm not sure that it's something that related to `jest-playwright`. Could you please try it with `playwright` only?

**Comment by devzerker** [comment]
@mmarkelov screenshots from expected behavior are from `playwright` only, here is code

```js
const playwright = require('playwright');

(async () => {
    const browser = await playwright.chromium.launch({ headless: false });
    let context = await browser.newContext();
    let page = await context.newPage();
    await page.goto('https://github.com');
    await page.type('input[name="q"]', 'Playwright');
    await page.press('input[name="q"]', 'Enter');
    await page.waitForTimeout(2000);
    context.close();

    context = await browser.newContext({ viewport: { width: 1600, height: 900 } });
    page = await context.newPage();
    await page.goto('https://github.com');
    await page.type('input[name="q"]', 'Playwright');
    await page.press('input[name="q"]', 'Enter');
    await page.waitForTimeout(2000);
    context.close();

    context = await browser.newContext({ ...require('playwright').devices['iPhone 11 Pro'] });
    page = await context.newPage();
    await page.goto('https://github.com');
    await page.waitForTimeout(2000);

    await browser.close();
})();

```

**Comment by devzerker** [comment]
Problem solved in playwright 1.3.0
